crates.io version bump request

https://docs.rs/humphrey_ws/latest/humphrey_ws/handler/fn.async_websocket_handler.html
This example doesn't compile when pulling from crates.io. I suspect it's just that crates.io needs a minor bump with
https://github.com/w-henderson/Humphrey/blob/3a5d2b5883fe62e5e89b4851cc9e08b41896d6c0/humphrey-ws/Cargo.toml#L17? I think `humphrey-auth` and `humprey-server` might also need a version bump.

Thank you very much for this project, this is exactly what I was looking for! I can dig deeper if it's not just a simple minor version bump.


Fails to compile with the following dependencies with no lock-file:
```
[dependencies]
humphrey = "^0.7.0"
humphrey_ws = "^0.5"
```
but it does work from a local clone of master:
```
humphrey = { path = "../Humphrey/humphrey" }
humphrey_ws = { path = "../Humphrey/humphrey-ws" }
```

error:
```
error[E0277]: expected a `Fn<(humphrey::http::Request, humphrey::stream::Stream, Arc<_>)>` closure, found `impl Fn(humphrey::http::request::Request, humphrey::stream::Stream, Arc<_>)`
   --> src/main.rs:14:38
    |
14  |         .with_websocket_route("/ws", async_websocket_handler(websocket_app.connect_hook().unwrap()));
    |          --------------------        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected an `Fn<(humphrey::http::Request, humphrey::stream::Stream, Arc<_>)>` closure, found `impl Fn(humphrey::http::request::Request, humphrey::stream::Stream, Arc<_>)`
    |          |
    |          required by a bound introduced by this call
    |
    = help: the trait `Fn<(humphrey::http::Request, humphrey::stream::Stream, Arc<_>)>` is not implemented for `impl Fn(humphrey::http::request::Request, humphrey::stream::Stream, Arc<_>)`
    = note: expected a closure with arguments `(humphrey::http::request::Request, humphrey::stream::Stream, Arc<_>)`
               found a closure with arguments `(humphrey::http::Request, humphrey::stream::Stream, Arc<_>)`
    = note: required for `impl Fn(humphrey::http::request::Request, humphrey::stream::Stream, Arc<_>)` to implement `humphrey::app::WebsocketHandler<_>`
note: required by a bound in `App::<State>::with_websocket_route`
   --> /home/grinkers/.cargo/registry/src/index.crates.io-6f17d22bba15001f/humphrey-0.7.0/src/app.rs:364:12
    |
362 |     pub fn with_websocket_route<T>(mut self, route: &str, handler: T) -> Self
    |            -------------------- required by a bound in this associated function
363 |     where
364 |         T: WebsocketHandler<State> + 'static,
    |            ^^^^^^^^^^^^^^^^^^^^^^^ required by this bound in `App::<State>::with_websocket_route`
    ```

--- a/humphrey-auth/Cargo.toml
+++ b/humphrey-auth/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "humphrey_auth"
-version = "0.1.4"
+version = "0.1.5"
 edition = "2021"
 license = "MIT"
 homepage = "https://github.com/w-henderson/Humphrey"

--- a/humphrey-server/Cargo.toml
+++ b/humphrey-server/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "humphrey_server"
-version = "0.6.0"
+version = "0.6.1"
 edition = "2018"
 license = "MIT"
 homepage = "https://github.com/w-henderson/Humphrey"

--- a/humphrey-ws/Cargo.toml
+++ b/humphrey-ws/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "humphrey_ws"
-version = "0.5.0"
+version = "0.5.1"
 edition = "2018"
 license = "MIT"
 homepage = "https://github.com/w-henderson/Humphrey"

